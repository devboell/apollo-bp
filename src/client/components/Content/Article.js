import styled from 'styled-components'

const Article = styled.article`
  display: flex;
  justify-content: flex-start;
  margin: 20px 50px;
  flex-basis: 80%;
`

export default Article
