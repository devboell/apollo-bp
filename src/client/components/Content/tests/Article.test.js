import React from 'react'
import 'jest-styled-components'

import Article from '../Article'

it('Article, styled component snapshot', () => {
  const wrapper = shallow(<Article />)
  expect(wrapper).toMatchSnapshot()
})
