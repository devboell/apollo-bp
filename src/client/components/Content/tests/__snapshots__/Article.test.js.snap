// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article, styled component snapshot 1`] = `
.egQGxY {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  margin: 20px 50px;
  -webkit-flex-basis: 80%;
  -ms-flex-basis: 80%;
  flex-basis: 80%;
}

<article
  className="sc-bdVaJa egQGxY"
/>
`;
